.github/copilot-instructions.md

```
@@ -0,0 +1,531 @@
+
+
+=== tailwindcss/v4 rules ===
+
+## Tailwind 4
+
+- Always use Tailwind CSS v4 - do not use the deprecated utilities.
+- `corePlugins` is not supported in Tailwind v4.
+- In Tailwind v4, you import Tailwind using a regular CSS `@import` statement, not using the `@tailwind` directives used in v3:
+
+<code-snippet name="Tailwind v4 Import Tailwind Diff" lang="diff"
+   - @tailwind base;
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)